### DISCUSSION TITLE
Return table rows on webhook call

### DISCUSSION THREAD

**Discussion by burnoutberni** [Budibase/budibase#1942] [Help] [answered]
Hi. I'm trying to build a thin Telegram bot layer around Budibase, so people can enter data into a form and that triggers some bot action. So far, so good.

Next step would be to retrieve data from Budibase, when someone replies to the bot. For that, I set up a new automation with a webhook trigger, but what action can I use? It seems like this is not possible, as long as I use the internal Budibase DB as a data storage. Do I have to move my data to another DB, so I can write a suitable query?

**Comment by mike12345567** [comment, marked as answer]
Hi @burnoutberni - currently it isn't possible to search from an automation, as you found you can perform queries to external data sources (like SQL) from automations but right now there isn't a good way to do this from an automation.

This is something that we will be touching on in the future, it is in our roadmap to give automations some general improvements, including creating actions like the ability to perform searches, iterate over an array of results etc.
